setup.cfg

```diff
@@ -13,7 +13,7 @@ install_requires =
 include_package_data = True
 
 [options.package_data]
-dss = manifests.yaml
+dss = manifest_templates/*
```

Comment:
This change was already merged to main please rebase.

Reply:
merged